tensorflow/compiler/xla/service/gpu/llvm_gpu_backend/gpu_backend_lib.cc

```diff
@@ -91,10 +92,18 @@ static string GetSmName(std::pair<int, int> compute_capability) {
       {{7, 5}, 75},
   });
   int sm_version = 35;
-  auto it = m->find(compute_capability);
-  if (it != m->end()) {
-    sm_version = it->second;
-  } else {
+  // If the current compute capability isn't known, fallback to the
+  // most recent version before the unknown version.
+  for (auto iter = m->begin(); iter != m->end(); ++iter) {
+    auto k = iter->first;
+    if ((k.first < compute_capability.first) ||
+	(k.first == compute_capability.first &&
+	 k.second <= compute_capability.second)) {
+      sm_version = iter->second;
+    }
+  }
+  std::cout << "SM " << sm_version;
```

Comment:
Debug print?  (Feel free to use a `VLOG(1)` for this.)

Reply:
Leftover debug. I removed it. I do not think it would be useful in other case then for this PR.